KhaiMai2016-PlanoTX.md

```diff
@@ -0,0 +1,7 @@
+# Plano, TX
+
+### Restaurants
```

Comment:
@KhaiMai2016 I will be in Plano next week, what is your favorite lunchtime restaurant